api-tests/platform/targets/tgt_dev_apis_tfm_nrf/nspe/pal_driver_intf.c

```diff
@@ -0,0 +1,148 @@
+/** @file
+ * Copyright (c) 2019-2020, Arm Limited or its affiliates. All rights reserved.
+ * SPDX-License-Identifier : Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+**/
+
+#include "pal_common.h"
+#include "pal_nvmem.h"
+#include <log/tfm_log.h>
+
+
+/**
+    @brief    - This function initializes the UART
+    @param    - uart base addr
+    @return   - SUCCESS/FAILURE
+**/
+int pal_uart_init_ns(uint32_t uart_base_addr)
+{
+    (void)uart_base_addr;
+    return PAL_STATUS_SUCCESS;
+}
+
+/**
+    @brief    - This function parses the input string and writes bytes into UART TX FIFO
+    @param    - str      : Input String
+              - data     : Value for format specifier
+    @return   - SUCCESS/FAILURE
+**/
+
+int pal_print_ns(const char *str, int32_t data)
+{
+    tfm_log_printf(str, data);
+    return PAL_STATUS_SUCCESS;
+}
+
+
+void tfm_plat_test_psa_timer_init(uint32_t ticks);
+void tfm_plat_test_psa_timer_start(void);
+void tfm_plat_test_psa_timer_stop(void);
```

Comment:
Wouldn't it be better to place all these declarations of external stuff right after inclusions? Especially `pal_nvmem_get_addr` is quite hidden.

Reply:
Done